client/state/ui/billing-transactions/actions.js

```diff
@@ -0,0 +1,72 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import {
+	BILLING_TRANSACTIONS_FILTER_SET_APP,
+	BILLING_TRANSACTIONS_FILTER_SET_MONTH,
+	BILLING_TRANSACTIONS_FILTER_SET_NEWEST,
+	BILLING_TRANSACTIONS_FILTER_SET_PAGE,
+	BILLING_TRANSACTIONS_FILTER_SET_QUERY,
+} from 'state/action-types';
+
+/**
+ * Sets the app filter on the given transactionType table
+ * @param {String} transactionType - transaction type: 'past' or 'upcoming'
+ * @param {String} app - app filter value
+ * @returns {Object} action
+ */
+export const setApp = ( transactionType, app ) => ( {
+	type: BILLING_TRANSACTIONS_FILTER_SET_APP,
+	transactionType,
+	app,
+} );
+
+/**
+ * Sets the date filter value on the given transactionType table to all transactions sorted by newest
+ * @param {String} transactionType - transaction type: 'past' or 'upcoming'
+ * @returns {Object} action
+ */
+export const setNewest = transactionType => ( {
```

Comment:
Could we use `setDate` for this one, and just define default `month` and `operator` to be `null` or `undefined`? Feels unnecessary to have a different action and action creator for something that simple.